apps/web/scripts/kernel/utils/shared.ts

```diff
@@ -0,0 +1,25 @@
+import { type MiniBroadcast } from '@walless/core';
+import { type MessagePayload, type PopupType } from '@walless/messaging';
+
+interface RequestContent {
+	channel: MiniBroadcast;
+	popup?: chrome.windows.Window;
+	payload: MessagePayload;
+	resolve: boolean;
+}
+
+export const requestMap: Record<string, RequestContent> = {};
```

Comment:
delete this `requestMap` its already defined in requestPool, rename to `popup.ts`, maybe